packages/ses/test/error/throws-and-logs.js

```diff
@@ -33,14 +26,7 @@ const compareLogs = freeze((t, log, goldenLog) => {
       ) {
         t.assert(logEntry instanceof goldenEntry, 'not the right error');
       } else {
-        // tap uses `===` instead of `Object.is`.
-        // Assuming ava does the right thing, switch back to this when
-        // switching back to ava.
-        // t.is(logEntry, goldenEntry);
-        t.assert(
-          is(logEntry, goldenEntry),
-          `${q(logEntry)} not same as ${q(goldenEntry)}`,
-        );
```

Comment:
`ava` does the right thing: https://github.com/avajs/ava/blob/0a050243ea13df959e6c1536b2513b5fe3557690/docs/03-assertions.md#isactual-expected-message